forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/KeyAware.kt

```diff
@@ -11,7 +11,9 @@ import java.security.interfaces.RSAPublicKey
 /**
  * Helper class to generate and sign the keys
  */
-class KeyAware(private var userId: String) {
+internal class KeyAware(private var userId: String) {
```

Comment:
Missing Copyright

Reply:
thanks , since i updated the android studio all my copyright generators are gone , need to fix it back